src/main/kotlin/no/nav/omsorgspenger/sak/HentOmsorgspengerSaksnummer.kt

```diff
@@ -30,10 +31,12 @@ internal class HentOmsorgspengerSaksnummer(
             .map { it.asText() }
             .toSet()
 
+        val historiskeIdenter = hentIdentPdlMediator.hentIdenter(identitetsnummer)
+
         logger.info("Løser behovet for ${identitetsnummer.size} personer.")
 
         val saksnummer = identitetsnummer
-            .map { it to hentSaksnummerFor(it) }
+            .map { it to hentSaksnummerFor(it, historiskeIdenter[it]) }
```

Comment:
hva med å bare sende inn et set? `historiskeIdentitetsnummer.plus(it)`

Reply:
Ah, vi fortfarande skilja på "original" ident genom att hämta last() ur settet, då går det att lagra nytt saksnummer på korrekt ident. Jag löser det imorgon.